holoviews/__init__.py

```diff
@@ -25,6 +25,7 @@
 from .operation import ElementOperation, MapOperation, TreeOperation # noqa (API import)
 from .element import *                                               # noqa (API import)
 from . import util # noqa (API import)
+from .element import __all__ as elements_list
```

Comment:
I know ``hv.element.__all__`` filters for ViewableElement but I think that includes some other types as well.